src/Features/OnboardingTasks/TaskLists.php

```diff
@@ -165,7 +165,8 @@ public static function get_task( $id, $task_list_id = null ) {
 		$tasks_to_search = $task_list ? $task_list['tasks'] : array_reduce(
 			self::get_lists(),
 			function ( $all, $curr ) {
-				return array_merge( $all, $curr['tasks'] );
+				$json = $curr->get_json();
+				return array_merge( $all, $json['tasks'] );
```

Comment:
@joelclimbsthings instead of converting it to `json` and then getting the tasks, should we instead expose either the `tasks` variable (making it public) or creating a `get_tasks` method within `TaskList.php`?

Reply:
Agreed.  Keep it in model form as long as possible before returning as JSON.

Reply:
Totally agreed @joshuatf @louwie17 . This led to doing more work on the endpoints than intended in this PR, but done in b37f537cbadb3dc2ff49ba08606b7f059281a004.